tests/functional/i/invalid/invalid_bases_assignment.py

```diff
@@ -0,0 +1,16 @@
+# pylint: disable=missing-module-docstring, missing-class-docstring
+# pylint: disable=too-few-public-methods
+
+
+class Apple:
+    pass
+
+
+Apple.__bases__ = ()  # [invalid-bases-assignment]
+Apple.__bases__ = "lala"  # [invalid-bases-assignment]
+Apple.__bases__ = True  # [invalid-bases-assignment]
+Apple.__bases__ = (1,)
```

Comment:
This one is a `TypeError`; perhaps we should make the [current definition](https://github.com/PyCQA/pylint/pull/7615/files#diff-9b76776694be02a09ca25a56adee0b3087ad5b86dfafd318923c4bf008c131c3R404) stricter.

Reply:
[Python docs](https://docs.python.org/3/c-api/object.html?highlight=__bases__) have it as _a tuple of base classes_

Reply:
good catch!